src/global_constants.jl

```diff
@@ -18,45 +18,46 @@ MᵣW_mul_factor(::True) = StaticInt{4}()
 MᵣW_mul_factor(::False) = StaticInt{9}()
 MᵣW_mul_factor() = MᵣW_mul_factor(has_feature(Val(:x86_64_avx512f)))
 
-
-if Sys.ARCH === :aarch64 && (Sys.isapple() || occursin("apple", Sys.CPU_NAME::String))
-  W₁Default() = StaticFloat64{0.23015506935919203}()
-W₂Default() = StaticFloat64{0.16967706087713014}()
-R₁Default() = StaticFloat64{0.9982516031563079}()
-R₂Default() = StaticFloat64{0.5167030291302886}()
+if Sys.ARCH === :aarch64 &&
+   (Sys.isapple() || occursin("apple", Sys.CPU_NAME::String))
+  W₁Default() = StaticFloat64{0.1464967933382345}()
+  W₂Default() = StaticFloat64{0.07243228432052636}()
+  R₁Default() = StaticFloat64{0.5024723443788641}()
+  R₂Default() = StaticFloat64{0.9018940596921994}()
```

Comment:
Those are the same as the current values.
Are you not seeing around 340ms from `Octavian.matmul!`?